src/main/java/graphql/execution/instrumentation/Instrumentation.java

```diff
@@ -42,34 +40,6 @@
  */
 @PublicSpi
 public interface Instrumentation {
-
-    /**
-     * This will be called just before execution to create an object that is given back to all instrumentation methods
-     * to allow them to have per execution request state
-     *
-     * @return a state object that is passed to each method
-     *
-     * @deprecated use {@link #createState(InstrumentationCreateStateParameters)} instead
-     */
-    @Deprecated(since = "2022-07-26")
-    default InstrumentationState createState() {
-        return null;
-    }
-
-    /**
-     * This will be called just before execution to create an object that is given back to all instrumentation methods
-     * to allow them to have per execution request state
-     *
-     * @param parameters the parameters to this step
-     *
-     * @return a state object that is passed to each method
-     */
-    @Deprecated(since = "2023-08-25")
-    @Nullable
-    default InstrumentationState createState(InstrumentationCreateStateParameters parameters) {
```

Comment:
2023-08-25 is not that long ago say for deprecation BUT we have to remove some methods for performance reasons so if we are going to make a breaking change - let's take the opportunity to remove the deprecated methods now anyway..